lib/valkyrie/persistence/solr/orm_converter.rb

```diff
@@ -32,32 +45,54 @@ def attributes
                            Valkyrie::Persistence::Attributes::OPTIMISTIC_LOCK => token)
     end
 
+    # Construct a Time object from the datestamp for the resource creation date indexed in Solr
+    # @return [Time]
     def created_at
       DateTime.parse(solr_document.fetch("created_at_dtsi").to_s).utc
     end
 
+    # Construct a Time object from the datestamp for the date of the last resource update indexed in Solr
+    # @return [Time]
     def updated_at
       DateTime.parse(solr_document["timestamp"] || solr_document.fetch("created_at_dtsi").to_s).utc
     end
 
+    # Construct the OptimisticLockToken object using the "_version_" field value in the Solr Document
+    # @see https://lucene.apache.org/solr/guide/7_4/updating-parts-of-documents.html#optimistic-concurrency
+    # @return [Valkyrie::Persistence::OptimisticLockToken]
     def token
       Valkyrie::Persistence::OptimisticLockToken.new(adapter_id: resource_factory.adapter_id, token: version)
     end
 
+    # Access the "_version_" field value within the Solr Document
+    # @return [String]
     def version
       solr_document.fetch('_version_', nil)
     end
 
+    # Retrieve the ID for the Valkyrie Resource in the Solr Document
+    # @note this assumes that the ID has been prepended with the string "id-"
+    # @return [String]
     def id
       solr_document.fetch('id').sub(/^id-/, '')
     end
 
+    # Construct the Hash containing the Valkyrie Resource attributes using the Solr Document
+    # @note this filters for attributes which have been indexed as stored multivalued texts (tsim)
+    # @see https://github.com/samvera-labs/valkyrie/blob/master/solr/config/schema.xml
+    # @see https://lucene.apache.org/solr/guide/7_4/defining-fields.html#defining-fields
```

Comment:
Same as above, do we want to use `https://lucene.apache.org/solr/guide/defining-fields.html#defining-fields` here?